src/Functions/FunctionsStringDistance.cpp

```diff
@@ -518,32 +518,206 @@ using FunctionJaroWinklerSimilarity = FunctionsStringSimilarity<FunctionStringDi
+
+    FunctionDocumentation::Description description_edit_utf8 = R"(
+Calculates the [edit distance](https://en.wikipedia.org/wiki/Edit_distance) between two UTF8 strings.
+)";
+    FunctionDocumentation::Syntax syntax_edit_utf8 = "editDistanceUTF8(s1, s2)";
+    FunctionDocumentation::Arguments arguments_edit_utf8 = {
+        {"s1", "First input string.", {"String"}},
+        {"s2", "Second input string.", {"String"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_edit_utf8 = {"Returns the edit distance between the two UTF8 strings.", {"UInt64"}};
+    FunctionDocumentation::Examples examples_edit_utf8 = {
+    {
+        "Usage example",
+        "SELECT editDistanceUTF8('我是谁', '我是我')",
+        R"(
+┌─editDistanceUTF8('我是谁', '我是我')──┐
+│                                   1 │
+└─────────────────────────────────────┘
+        )"
+    }
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_utf8 = {24, 6};
+    FunctionDocumentation documentation_edit_utf8 = {description_edit_utf8, syntax_edit_utf8, arguments_edit_utf8, returned_value_edit_utf8, examples_edit_utf8, introduced_in_utf8, category};
+
+    FunctionDocumentation::Description description_damerau = R"(
+Calculates the [Damerau-Levenshtein distance](https://en.wikipedia.org/wiki/Damerau%E2%80%93Levenshtein_distance) between two byte strings.
+)";
+    FunctionDocumentation::Syntax syntax_damerau = "damerauLevenshteinDistance(s1, s2)";
+    FunctionDocumentation::Arguments arguments_damerau = {
+        {"s1", "First input string.", {"String"}},
+        {"s2", "Second input string.", {"String"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_damerau = {"Returns the Damerau-Levenshtein distance between the two strings.", {"UInt64"}};
+    FunctionDocumentation::Examples examples_damerau = {
+    {
+        "Usage example",
+        "SELECT damerauLevenshteinDistance('clickhouse', 'mouse')",
+        R"(
+┌─damerauLevenshteinDistance('clickhouse', 'mouse')─┐
+│                                                 6 │
+└───────────────────────────────────────────────────┘
+        )"
+    }
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_damerau = {24, 1};
+    FunctionDocumentation documentation_damerau = {description_damerau, syntax_damerau, arguments_damerau, returned_value_damerau, examples_damerau, introduced_in_damerau, category};
+
+    FunctionDocumentation::Description description_jaccard = R"(
+Calculates the [Jaccard similarity index](https://en.wikipedia.org/wiki/Jaccard_index) between two byte strings.
+)";
+    FunctionDocumentation::Syntax syntax_jaccard = "stringJaccardIndex(s1, s2)";
+    FunctionDocumentation::Arguments arguments_jaccard = {
+        {"s1", "First input string.", {"String"}},
+        {"s2", "Second input string.", {"String"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_jaccard = {"Returns the Jaccard similarity index between the two strings.", {"Float64"}};
```

Comment:
\+ wikipedia link

Reply:
Linked above in the description, do we need it in the return value as well?

Reply:
Nope.